Package.swift

```diff
@@ -0,0 +1,30 @@
+// swift-tools-version: 5.8
+
+import PackageDescription
+
+private let packageName = "SimpleNetworkService"
+private let packageTestName = "SimpleNetworkServiceTests"
+
+let package = Package(
+    name: packageName,
+    platforms: [
+        .iOS(.v15)
+    ],
+    products: [
+        .library(
+            name: packageName,
+            targets: [packageName]
+        )
+    ],
+    dependencies: [],
+    targets: [
+        .target(
+            name: packageName,
```

Comment:
Can we give the target more specific name? Like `NetworkService`?

There's no requirement that the target name should be equal to the package's name. Make it readable and easy-to-understand please.

Reply:
fixed